share/tasks.py

```diff
@@ -44,7 +44,12 @@ def transform(self, raw_id):
         graph = transformer.transform(raw)
 
         if not graph or not graph['@graph']:
-            logger.warning('Graph was empty for %s, skipping...', raw)
+            if not raw.normalizeddata_set.exists():
+                logger.warning('Graph was empty for %s, setting no_change to True', raw)
+                RawDatum.objects.filter(id=raw_id).update(no_change=True)
```

Comment:
This is the only time that rawdata could be processed and not generate a normalized data, the case that this boolean exists to handle.
If there is not a way to detect this and we ever have more dud rawdata than `limit` this task will no longer do anything.

My original explanation is definitely wrong, however, as a normalized data would still be created in that case there is no need for that flag (currently, it would be very helpful).